test/integration/data_bags/x86_db_passwords/placement.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "placement",
+  "placement": {
+    "encrypted_data": "m3Lz5HMrn+naTm+YOgJ00lNDB38+mgeM7fyjMtGYByU=\n",
+    "hmac": "DXYla64sQ7PK1p2ZPj1O7SPquKeeRbqUGJTgi8/CAL0=\n",
+    "iv": "eiEYjxWXt+R9v4IYc5oyDA==\n",
+    "version": 2,
+    "cipher": "aes-256-cbc"
+  }
+}
```

Comment:
Add newline

Reply:
I think, unless json cant have it.

Reply:
The encrypted data bag does it this way so it's OK to leave it as-is.

Reply:
:+1: